upstairs/src/active_jobs.rs

```diff
@@ -354,7 +354,9 @@ impl BlockMap {
         // We record every job, even if it's empty
         self.job_to_range.insert(job, r);
 
-        let Some(r) = Self::blocks_to_range(r) else { return; };
+        let Some(r) = Self::blocks_to_range(r) else {
```

Comment:
It is. I needed to bump the toolchain to deal with what appears to be a bug in 1.70. If you run `cargo update -p oximeter` on `main` using the existing pinned toolchain, you'll see:

```console
bnaecker@shale : ~/crucible $ cargo update -p oximeter
    Updating git repository `https://github.com/oxidecomputer/omicron`
    Updating crates.io index
    Updating git repository `https://github.com/oxidecomputer/dropshot`
    Updating git repository `https://github.com/oxidecomputer/progenitor`
error: no matching package named `omicron-common` found
location searched: https://github.com/oxidecomputer/omicron?branch=main
required by package `crucible-agent v0.0.1 (/home/bnaecker/crucible/agent)`
```

I am not sure what exactly is going on here, and was going to raise bumping the toolchain as a "rider" on this PR in case there are strong opinions against doing that.